### PR TITLE
Add Tofino to the registry as a handler for html and http and parse urls passed on the command line.

### PR DESCRIPTION
**Pull request by Mossop**
This adds Tofino as a browser to the windows registry on install and removes on uninstall. It's a bit clunky, you get a UAC escalation request from regedit but it works for now. After that you can choose tofino in the windows default apps list.

The command line handling is a bit clunky too. It might be nice to rework that at some point but it works for now.

Currently as long as the app is already open links open in a new tab. If the app isn't open it fails because of #1321.

### CODE REVIEW

File: app/main/index.js [hunk 1]
```diff
@@ -71,6 +74,24 @@ switch (seenEvent) {
     require('./browser');
     break;
 
+  case 'squirrel-install':
+    addToWindowsRegistry().then(() => {
```
Comment (jsantell) [inline app/main/index.js]: You can call the promise method `catch(fn)` which is like `then(null, fn)`

File: package.json [hunk 2]
```diff
@@ -60,6 +60,7 @@
     "redux-logger": "2.7.0",
     "redux-saga": "0.12.0",
     "source-map-support": "ncalexan/node-source-map-support#fileUrls-plus",
+    "tmp": "0.0.29",
```
Comment (jsantell) [inline package.json]: This is node-only, should it live in app/package.json?

File: app/package.json [hunk 3]
```diff
@@ -12,7 +12,9 @@
   "license": "Apache-2.0",
   "dependencies": {
     "datomish-user-agent-service": "0.0.2",
+    "fs-promise": "0.5.0",
     "request": "2.65.0",
+    "thenify": "3.2.0",
```
Comment (victorporof) [inline app/package.json]: Potentially unrelated, I don't understand when we should add dependencies to app/package.json exactly. Is there a reason why these aren't in the top level package.json?

Comment (victorporof) [inline app/package.json]: Should we migrate some of the top level dependencies here instead? Or vice-versa?

Comment (Mossop) [inline app/package.json]: I wish we could add comments to the json to explain them. It's basically for packages that webpack fails to package for some reason. In this case thenify relies on any-promise which attempts to require a whole bunch of promise modules that we don't have installed which was causing webpack to fail.

Comment (victorporof) [inline app/package.json]: Same for fs-promise?

Comment (Mossop) [inline app/package.json]: Yes, it requires any-promise too.

**Pull request merged**